moveit_plugins/moveit_ros_control_interface/CMakeLists.txt

```diff
@@ -1,101 +1,85 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_control_interface)
 
-if(NOT "${CMAKE_CXX_STANDARD}")
-  set(CMAKE_CXX_STANDARD 14)
-endif()
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-find_package(catkin REQUIRED COMPONENTS
-  actionlib
-  controller_manager_msgs
-  moveit_core
-  moveit_simple_controller_manager
-  pluginlib
-  trajectory_msgs
-)
-## System dependencies are found with CMake's conventions
-find_package(Boost REQUIRED COMPONENTS system thread)
-
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES
-    ${PROJECT_NAME}_plugin
-    ${PROJECT_NAME}_trajectory_plugin
-  CATKIN_DEPENDS
-    actionlib
+find_package(ament_cmake REQUIRED)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+    rclcpp_action
     controller_manager_msgs
     moveit_core
+    moveit_simple_controller_manager
+    pluginlib
     trajectory_msgs
-  DEPENDS Boost
 )
 
-include_directories(include)
-include_directories(SYSTEM ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+foreach(dependency IN ITEMS ${THIS_PACKAGE_INCLUDE_DEPENDS})
```

Comment:
This is a really nice way of using the include depends. Maybe a best practice we want to apply everywhere?